src/components/Selector.jsx

```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import "../styles/Selector.scss";
 
 function Select({ value, onChange, options }) {
   return (
-    <select onChange={onChange} value={value}>
+    <select onChange={onChange} value={value} className="selector">
```

Comment:
Same as the file name
```suggestion 
    <select onChange={onChange} value={value} className="select">
```

Reply:
done :)